sycl/include/CL/sycl/detail/pi.h

```diff
@@ -38,6 +38,8 @@
 // 4. Add interoperability interfaces for kernel.
 // 4.6 Added new ownership argument to piextQueueCreateWithNativeHandle which
 // changes the API version from 3.5 to 4.6.
+// 5.7 Added new context and ownership arguments to
+//   piextEventCreateWithNativeHandle
 //
 #include "CL/cl.h"
 #define _PI_H_VERSION_MAJOR 4
```

Comment:
Should the major or minor version be changed?

Reply:
Sure, I've added comment about it but missed to change here